submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,99 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  friendsToString() {
+    return this.friends.join(", ");
+  }
+
+  output() {
+    const { name, species, gender, saying } = this;
+
+    return `${name}; ${species}; ${gender}; ${saying}; ${this.friendsToString()}`;
```

Comment:
Let's employ `Array#join`. Change this line only.
Remember, anything can be an array element. `[ func() ]` will add a result from func call as an array element.